src/v2/view-builder/internals/BaseForm.ts

```diff
@@ -164,25 +165,24 @@ export default Form.extend({
     const messages = this.options.appState.get('messages') || {};
     const errContainer = '.o-form-error-container';
     if (Array.isArray(messages.value) && !(options instanceof View)) {
-      this.add('<div class="ion-messages-container"></div>', errContainer);
-      messages.value.forEach(obj => {
-        if(!obj?.class || obj.class === INFO_MESSAGE_CLASS) {
-          // add message as plain text
-          this.add(`<p>${obj.message}</p>`, '.ion-messages-container');
```

Comment:
TestCafe 3 produces error
```
Cannot obtain information about the node because the specified selector does not match any node in the DOM tree.
      
         | Selector('.o-form')
         |   .nth(0)
       > |   .find('.ion-messages-container')
```
Can be fixed if `ion-messages-container` div html is constructed at once (or if messages are added with `this.add` after small timeout which is not desired)

Reply:
32 tests are affected by this issue:
[See full list of affected tests on Bacon](https://bacon-go.aue1e.saasure.net/commits?artifact=okta-signin-widget&branch=od-unquarantine-poll-tests-OKTA-796307&page=1&pageSize=6&sha=c1568ad9bef86626bed8633a1d5b7178bed62088&tab=tests&testSuite=okta-signin-widget-testcafe)

[Disabling native automation](https://testcafe.io/documentation/402638/reference/configuration-file#disablenativeautomation) solves this issue.

So seems like the best way to solve the issue without modifying source code is to disable native automation

Reply:
Error is produced at this line
https://github.com/okta/okta-signin-widget/blob/master/src/v2/view-builder/internals/BaseForm.ts#L171
https://github.com/okta/okta-signin-widget/blob/master/packages/%40okta/courage-dist/esm/src/courage/framework/View.js#L387